src/components/Atoms/Shapes/ColoredCircle/ColoredCircle.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import {
+  StyleSheet,
+  TouchableOpacity,
+} from 'react-native'
+
+import gStyle from '../../../styles'
+
+const size = 160
```

Comment:
seems sensible to also allow size to come in through props

Reply:
🏁